lib/inspec/runner_rspec.rb

```diff
@@ -98,13 +99,30 @@ def reset
 
     private
 
-    FORMATTERS = {
-      'json-min' => 'InspecRspecMiniJson',
-      'json' => 'InspecRspecJson',
-      'json-rspec' => 'InspecRspecVanilla',
-      'cli' => 'InspecRspecCli',
-      'junit' => 'InspecRspecJUnit',
-    }.freeze
+    # Set optional formatters and output
+    #
+    #
+    def set_optional_formatters
+      return if @conf[:reporter].nil?
+      if @conf[:reporter].key?('json-rspec')
+        if @conf[:reporter]['json-rspec'].nil?
+          RSpec.configuration.add_formatter(Inspec::Formatters::RspecJson)
```

Comment:
We could probably use an explanation comment here and at line 119 explaining why we can't pass a second argument of `nil` - i.e. what happens if we do that?